app/models/sample.rb

```diff
@@ -848,4 +853,13 @@ def metadata_fields_info
   def first_pipeline_run
     pipeline_runs.order(created_at: :desc).first
   end
+
+  def status_url
+    base_url = if Rails.env == 'staging'
```

Comment:
What happens in dev?

Reply:
I know this was copied from pipeline run, but this still seems like the wrong place to define the url. This can probably be a constant defined in a shareable place. Can this be obtained from routes?

Reply:
If you still have access to the request when triggering the job, you can send it to the resque job from the original request using something like `request.base_url`...

Reply:
> What happens in dev? 

The link will be to prod which may be inaccurate. I could update it to point to localhost. 

> Can this be obtained from routes?

Probably. I would just need to research it.